### PR TITLE
documented how to save bytes in browsers

### PR DESCRIPTION
**Pull request by bevacqua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,15 @@ atob('SGVsbG8sIHdvcmxkIQ=='); // 'Hello, world!'
 
 [Per the spec](https://html.spec.whatwg.org/multipage/webappapis.html#atob:dom-windowbase64-btoa-3), `btoa` will accept strings "containing only characters in the range `U+0000` to `U+00FF`." If passed a string with characters above `U+00FF`, `btoa` will return `null`. If `atob` is passed a string that is not base64-valid, it will also return `null`. In both cases when `null` is returned, the spec calls for throwing a `DOMException` of type `InvalidCharacterError`.
 
+## Browsers
+
+If you want to include just one of the methods to save bytes in your client-side code, you could `require` the desired module directly.
+
+```js
+var atob = require('abab/lib/atob')
```
Comment (jeffcarp) [inline README.md]: Since this repo's style uses semicolons, would you mind adding them to these examples?

**Comment by jeffcarp** [comment]
This is awesome, thank you @bevacqua! I left one nitpick but other than that, LGTM :+1:

**Comment by bevacqua** [comment]
Fixed. Sorry about that :rocket:

**Comment by jeffcarp** [comment]
No worries, thanks for being lightning fast! I'm running out right now but I'll merge this tonight.

**Comment by jeffcarp** [comment]
Thanks again @bevacqua! Merged as 3f9daf5e17c66b4878ac7861b9c9c82724f03ce4

**Pull request closed without merging**